flake.nix

```diff
@@ -3,19 +3,20 @@
 
   inputs = {
     nixpkgs.follows = "haskellNix/nixpkgs-unstable";
-    hackageNix = {
-      url = "github:input-output-hk/hackage.nix";
-      flake = false;
-    };
+    utils.url = "github:numtide/flake-utils";
     haskellNix = {
       url = "github:input-output-hk/haskell.nix";
       inputs.hackage.follows = "hackageNix";
     };
-    utils.url = "github:numtide/flake-utils";
+    hackageNix = {
+      url = "github:input-output-hk/hackage.nix";
+      flake = false;
+    };
     iohkNix = {
       url = "github:input-output-hk/iohk-nix";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+    devxShell.url = "github:input-output-hk/devx";
```

Comment:
I'm still unsure if we want to _embed_ this or tell people to just go directly `nix develop github:input-output-hk/devx#...`?

Reply:
OK, I thought this would make it easier, but then we'll also have to keep it up-to-date. I'll remove this for the time being and we can always add it back if we change our minds